source/include/BuildConfig.h

```diff
@@ -244,6 +232,22 @@ SDG
 
 #endif
 
+// TODO(PaulAustin): allow for thread locals on linux/unix
+#define VIVM_THREAD_LOCAL
+
+#ifdef VIREO_DEBUG
+#define VIREO_TRACK_MEMORY_QUANTITY
+#define VIREO_USING_ASSERTS
+#else
+#if (kVireoOS_macosxU || kVireoOS_linuxU || kVireoOS_windows)
```

Comment:
I did this because I want local builds by default to also have asserts, and it's also nice if the CI build has asserts but is still optimized -- it has no use for debugging info, and runs slower with a debug build, and we really do want to test the optimizer in at least one of the CIs since that frequently finds bugs that don't show up in debug builds, yet the asserts still catch real issues.  

If you'd like, I could add a separate flag besides DEBUG that lets you turn on asserts independently and use that instead (as well as making it the default for local builds).  (Btw, the LLVM build a Release+Asserts build option, so there's precedent. :-) )

Reply:
Ok, I've made the enabling of asserts in a non-debug build explicit by pre-defining VIREO_USING_ASSERTS in the Makefile (instead of in code when DEBUG is off)  (commit 1deb65).